assets/node.yaml

```diff
@@ -10,6 +10,10 @@ spec:
     matchLabels:
       app: openstack-manila-csi
       component: nodeplugin
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
```

Comment:
`RollingUpdate` with `maxUnavailable: 1` is the default.  [The bug][1] is asking for `maxUnavailable: 10%`, so rollout time remains bounded as the node count grows.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1933184#c0

Reply:
Fixed